src/index.html

```diff
@@ -1,14 +1,28 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <!--   <h1>Moyo header</h1> -->
+    <header>
+      <nav class="container">
+        <a class="category is-active" href="">АРРLE</a>
+        <a class="category samsung" href="">SAMSUNG</a>
+        <a class="category phones" href="">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+        <a class="category" data-qa="hover"
+          href="">НОУТБУКИ И КОМПЬЮТЕРЫ
+        </a>
+        <a class="category" href="">ГАДЖЕТЫ</a>
```

Comment:
Не оставляй href пустым
```suggestion
        <a class="category" href="#">ГАДЖЕТЫ</a>
```

Reply:
Добавил localhost

Reply:
Можно просто `#`